nipype/interfaces/traits_extension.py

```diff
@@ -218,6 +218,38 @@ def __init__(self, value='', auto_set=False, entries=0,
         super(Directory, self).__init__(value, auto_set, entries, exists,
                                         **metadata)
 
+class ImageFile(File):
+    """ Defines a trait of specific neuroimaging files """
+
+    def __init__(self, value='', filter=None, auto_set=False, entries=0,
+                 exists=False, types=['nii', 'hdr', 'img'], compressed=True,
```

Comment:
link to nifti1 vs nifti2?

maybe we can work around this by adding a field to `ImageFile` that specifies `format_version`?